rpc/client.go

```diff
@@ -58,27 +58,26 @@ func (sc *Client) Close() {
 // ChainID retrieves the current chain ID for transaction replay protection.
 func (sc *Client) ChainID(ctx context.Context) (string, error) {
 	var result string
-	err := sc.c.CallContext(ctx, &result, "starknet_chainId")
+	err := sc.c.CallContext(ctx, &result, "starknet_chainId", []interface{}{}...)
```

Comment:
That is the solution I have found to address #66 . TL;DR is that we would have to talk to the people at Shardlabs to see if that is a bug and can be corrected. At least I did not face the issue with Pathfinder.